images/base/Dockerfile.centos7

```diff
@@ -4,9 +4,9 @@
 #
```

Comment:
I think merging them would be great.
I am assuming that "merging them" means have the regular Dockerfile look like Dockerfile.centos7 and not the other way around.